codalab/server/bundle_manager.py

```diff
@@ -826,7 +826,9 @@ def _get_staged_bundles_to_run(self, workers, user_info_cache):
                     # Remove the uuid from self._bundles_without_matched_workers if a matched
                     # private worker is found in the system and update bundle's metadata
                     if bundle.uuid in self._bundles_without_matched_workers:
-                        self._model.update_bundle(bundle, {'metadata': {'staged_status': None}})
+                        self._model.update_bundle(
```

Comment:
Why is this related to `cl edit`?  Should be in a different PR?

Reply:
`cl edit` doesn't work properly with the above string decoding fix. When a user types `cl edit <uuid>` to edit metadata fields in the pop-up window, if any of the fields are empty, CLI will just return a metadata key is missing as follows, as the following field has value `None`, which was treated as a deletion operation in update function.
```
stanford@Stanfords-MacBook-Pro codalab-worksheets % cl edit 0x075eaa877bf84ea68072995b19891474  
UsageError: Missing metadata key: allow_failed_dependencies
```